examples/cluster/plot_cluster_comparison.py

```diff
@@ -42,21 +42,34 @@
 blobs = datasets.make_blobs(n_samples=n_samples, random_state=8)
 no_structure = np.random.rand(n_samples, 2), None
 
+# Anisotropicly distributed data
+random_state = 170
+X, y = datasets.make_blobs(n_samples=n_samples, random_state=random_state)
+transformation = [[0.6, -0.6], [-0.4, 0.8]]
+X_aniso = np.dot(X, transformation)
+aniso = (X_aniso,y)
+
+# Different variance
+varied = datasets.make_blobs(n_samples=n_samples,
+                                cluster_std=[1.0, 2.5, 0.5],
+                                random_state=random_state)
+
```

Comment:
Align cluster_std and random_state with n_samples.
Use flake8 to check those things.